### ISSUE TITLE
Placeholder not visible for disabled multi-select

### ISSUE THREAD

**Issue by amankasliwal** [angular-ui/ui-select#1044]
When using Multiselect in Select2 theme, the place holder gets hidden when `ng-disabled="true"`
While inspecting, found that the place holder attribute is being written inside a `<span>` tag which doesn't work.

`<span class="ui-select-match" placeholder="Select options."></span>`

**Comment by dtagdev1** [comment]
me too

**Comment by jziggas** [comment]
Posted a fix in #1156
> This seemed to work for me:
>
```
.ui-select-multiple.ui-select-bootstrap > div > input.ui-select-search[disabled] {
    display: initial !important;
    cursor: not-allowed;
}
```

**Comment by user378230** [comment]
Closed by #1717
